docs/OVERVIEW.md

```diff
@@ -52,14 +52,22 @@ The root of this file is defined as:
         ...
     },
 
+    // There are shortcuts available to make component converting. The shortcuts are 'ctrl' or 'cmd' + numbers from 0 to 9.
+    // Here is a list of components which should be binded to the shortcuts, maximum is 10 components.
+    "conversionShortcutComponents": [
```

Comment:
Sorry I'm here :) But you have used "shortcuts" -> "conversionComponents", then here should be too
```suggestion
"shortcuts": {
    "conversionComponents": [
```

Reply:
You are always welcome and correct :)
I forgot to search over md files also.